resources/lang/en/consent.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Consent
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used for consent screens.
+    |
+    */
+
+    'agree' => 'Agree',
+    'disagree_logout' => 'Disagree and logout',
+    'skip' => 'Skip this step',
+    'save' => 'Save my preferences',
+
+    'privacy' => [
+        'dialog_title' => 'We need your confirmation',
+        'dialog_description' => 'Please review carefully our new privacy policy including all information on how we treat and your rights related to your data.',
+        'show_full_text' => 'Show me the full legal text',
+    ],
+    
+    'others' => [
+        'dialog_title' => 'One last thing from you',
+        'dialog_description' => 'The K-Box offers different services, we want to get your preferences about E-Mail notifications and statistics collection.',
+    ],
+    
+    'notification' => [
+        'dialog_title' => 'E-Mail notifications',
+        'dialog_description' => 'Would you like to receive notifications per E-Mail?',
```

Comment:
`Receive notifications per Email of any interactions of other users with you and your documents.`